### PR TITLE
template

### PR DESCRIPTION
**Pull request by frankiebee**
The purpose of this PR is to document our new file-structure pattern

### CODE REVIEW

**Comment by mixmix** [comment]
I wupped your wip. It's getting there

File: src/_template/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  command
+  interaction
+
+  constants:::optional
+  utils:::optional
+  
+  main --> command & interaction
+end
+
+entropy-base --> main
```
Comment (frankiebee) [inline src/_template/README.md:26]: does not show up in dark mode

File: src/_template/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Domain Template
+
+This folder described how we structure our "Domain" folders.
+
+```mermaid
```
Comment (frankiebee) [inline src/_template/README.md:5]: give this whole view a white background

File: src/_template/README.md (hunk 1 above)
Comment (mixmix) [reply src/_template/README.md:25]: Fixed

File: src/_template/README.md (hunk 2 above)
Comment (mixmix) [reply src/_template/README.md:5]: Can't change the whole background that I can see. This is a Github thing

**Review by rh0delta — APPROVED** [review]
looks good for now

**Pull request merged**